define/define.py

```diff
@@ -0,0 +1,36 @@
+import urllib2
+import json
+import random
+
+def get_rhyme(word):
+    request = urllib2.urlopen('https://api.datamuse.com/words?rel_rhy=' + word)
+    results = json.loads(request.read().decode("utf-8"))
+    return str(random.choice(results)['word'])
+
+def define_word(word):
+    request = urllib2.urlopen('http://api.pearson.com/v2/dictionaries/ldoce5/entries?headword=' + word)
+    results = json.loads(request.read().decode("utf-8"))
+    entry = random.choice(results['results'])
+    definition = random.choice(entry['senses'][0]['definition'])
+    return str(definition)
+
+def define_command(args):
+    word = args.split()[0]
+    rhyme_word = get_rhyme(word)
```

Comment:
Since Datamuse sometimes returns the original word, you may want to handle that case.